src/main/java/executor/command/Command.java

```diff
@@ -6,6 +6,7 @@
 public abstract class Command {
     protected Boolean exitRequest = false;
     protected CommandType commandType;
+    protected String description = "Basic";
```

Comment:
Maybe we can change it to "MISSING DESCRIPTION" so if any of us runs the HELP command its very clear its missing.